osbenchmark/client.py

```diff
@@ -93,13 +105,34 @@ def restore_context(cls, token):
     def update_request_start(cls, new_request_start):
         meta = cls.request_context.get()
         # this can happen if multiple requests are sent on the wire for one logical request (e.g. scrolls)
-        if "request_start" not in meta:
```

Comment:
@saimedhi @IanHoang is this backward compatible?

Reply:
@VijayanB, yes.